tests/unit/models/reddit/test_subreddit.py

```diff
@@ -123,3 +123,10 @@ def test__getitem(self):
         wikipage = subreddit.wiki["Foo"]
         assert isinstance(wikipage, WikiPage)
         assert "foo" == wikipage.name
+
+
+class TestSubredditModmailConversationsStream(UnitTest):
+    def test_random(self):
```

Comment:
This method name doesn't seem descriptive of the test's behavior to me, though perhaps I'm missing something. Can you comment?

Reply:
It was suggested by @PokestarFan to achieve 100% coverage. Not sure why it isn't 100% without it and what to name it. Will come up with a better name next commit.